SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -143,16 +145,12 @@ struct FWorkerTypeLaunchSection
 	UPROPERTY(Category = "SpatialGDK", EditAnywhere, config, meta = (EditCondition = "bLoginRateLimitEnabled"))
 	FLoginRateLimitSection LoginRateLimit;
 
-	/** Number of columns in the rectangle grid load balancing config. */
-	UPROPERTY(Category = "SpatialGDK", EditAnywhere, config, meta = (DisplayName = "Rectangle grid column count", ClampMin = "1", UIMin = "1"))
-	int32 Columns;
-
-	/** Number of rows in the rectangle grid load balancing config. */
-	UPROPERTY(Category = "SpatialGDK", EditAnywhere, config, meta = (DisplayName = "Rectangle grid row count", ClampMin = "1", UIMin = "1"))
-	int32 Rows;
+	/** Automatically or manually specifies the number of worker instances to launch in editor. */
+	UPROPERTY(Category = "SpatialGDK", EditAnywhere, config, meta = (DisplayName = "Automatically compute number of instances to launch in Editor"))
+	bool bAutoNumEditorInstances;
 
 	/** Number of instances to launch when playing in editor. */
-	UPROPERTY(Category = "SpatialGDK", EditAnywhere, config, meta = (DisplayName = "Instances to launch in editor", ClampMin = "0", UIMin = "0"))
+	UPROPERTY(Category = "SpatialGDK", EditAnywhere, config, meta = (DisplayName = "Instances to launch in editor", ClampMin = "0", UIMin = "0", EditCondition = "!bAutoNumEditorInstances"))
 	int32 NumEditorInstances;
```

Comment:
Can this adapt dynamically? It seems like there's only a benefit when the editor is first launched (for me I had quadrant strat and it set this to 4 which was correct). When I updated the # of rows in the strat the # of instances remained 4 which seems unintended?

Reply:
This is intended as a debug option, so I do not think the added complexity is worth it.

Reply:
What I'm saying is, I'm not sure there's benefit in having an automatic setting if it doesn't adapt to other things changing? Setting it initially basically on what other settings are at load time is good, but we could do that with leaving the manual field as it is?

Reply:
Odd, it should not be automatically populated at all. Maybe it just retained its previous value from conf since I did not rename the field ?